### PR TITLE
Feature/home page UI

### PR DESCRIPTION
**Pull request by JRobbo85**
@mz106 please help with merge conflict

### CODE REVIEW

**Comment by JRobbo85** [comment]
@mz106 merge fixed needs review

File: src/App.js [hunk 1]
```diff
@@ -15,11 +17,9 @@ function App() {
   }, []);
 
   return (
-    <>
-      <h1>hello world</h1>
-      <h2>Branch protection test</h2>
-    </>
-  );
+
```
Comment (mz106) [inline src/App.js:20]: please leave in placeholder text until ready to change

File: src/pages/home/Home.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#homeLayout{
+    display:flex;
+    flex-direction: column;
+}
+
+#cardRow{
+    display:flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
```
Comment (mz106) [inline src/pages/home/Home.css]: Please add empty line

File: src/pages/home/Home.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import './Home.css';
+
+
+const HomeLayout = () => {
```
Comment (mz106) [inline src/pages/home/Home.js]: Please edit after conversation

File: src/pages/home/Home.css (hunk 2 above)
Comment (mz106) [inline src/pages/home/Home.css]: Please add new line

**Review by mz106 — CHANGES_REQUESTED** [review]
Please review single comments

**Review by mz106 — APPROVED** [review]
Approved

**Comment by mz106** [comment]
All good to merge @JRobbo85

**Pull request merged**